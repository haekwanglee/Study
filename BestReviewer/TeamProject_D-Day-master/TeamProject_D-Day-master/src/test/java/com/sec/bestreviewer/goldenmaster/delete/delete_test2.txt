SCH,-p,-f,-se,name,C
DEL, ,-f, ,name,C
SCH,-p,-f,-se,name,C
SCH,-p, ,-se,certi,EX