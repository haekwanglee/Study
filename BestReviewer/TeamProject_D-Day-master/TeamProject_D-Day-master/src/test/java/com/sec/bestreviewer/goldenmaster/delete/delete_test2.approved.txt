SCH,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,ADV
SCH,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,ADV
SCH,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,ADV
SCH,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,ADV
SCH,92050301,AAAAA DDD,CL2,010-1111-1116,19980106,ADV
DEL,NONE
SCH,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,ADV
SCH,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,ADV
SCH,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,ADV
SCH,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,ADV
SCH,92050301,AAAAA DDD,CL2,010-1111-1116,19980106,ADV
SCH,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
SCH,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,ADV
SCH,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,ADV
SCH,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,ADV
SCH,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,ADV
